build broken on clang 9.1.0 on macOS

subj:
```
 >> 3484    Undefined symbols for architecture x86_64:
     3485      "dealii::VectorizedArray<double>::n_array_elements", referenced from:
     3486          dealii::MatrixFree<1, double>::n_active_entries_per_cell_batch(unsigned int) const in matrix_free.cc.o
     3487          dealii::MatrixFree<1, double>::n_active_entries_per_face_batch(unsigned int) const in matrix_free.cc.o
     3488          dealii::MatrixFree<2, double>::n_active_entries_per_cell_batch(unsigned int) const in matrix_free.cc.o
     3489          dealii::MatrixFree<2, double>::n_active_entries_per_face_batch(unsigned int) const in matrix_free.cc.o
     3490          dealii::MatrixFree<3, double>::n_active_entries_per_cell_batch(unsigned int) const in matrix_free.cc.o
```

in reference to https://github.com/dealii/dealii/pull/6720
@masterleinad ping.

--- a/include/deal.II/base/vectorization.h
+++ b/include/deal.II/base/vectorization.h
@@ -162,7 +162,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 1;
+  static const unsigned int n_array_elements = 1;
 
   // POD means that there should be no user-defined constructors, destructors
   // and copy functions (the standard is somewhat relaxed in C++2011, though).
@@ -568,7 +568,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 8;
+  static const unsigned int n_array_elements = 8;
 
   /**
    * This function can be used to set all data fields to a given scalar.
@@ -970,7 +970,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 16;
+  static const unsigned int n_array_elements = 16;
 
   /**
    * This function can be used to set all data fields to a given scalar.
@@ -1405,7 +1405,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 4;
+  static const unsigned int n_array_elements = 4;
 
   /**
    * This function can be used to set all data fields to a given scalar.
@@ -1784,7 +1784,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 8;
+  static const unsigned int n_array_elements = 8;
 
   /**
    * This function can be used to set all data fields to a given scalar.
@@ -2194,7 +2194,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 2;
+  static const unsigned int n_array_elements = 2;
 
   /**
    * This function can be used to set all data fields to a given scalar.
@@ -2532,7 +2532,7 @@ public:
   /**
    * This gives the number of vectors collected in this class.
    */
-  static constexpr unsigned int n_array_elements = 4;
+  static const unsigned int n_array_elements = 4;
 
   /**
    * This function can be used to set all data fields to a given scalar.

--- a/source/base/vectorization.cc
+++ b/source/base/vectorization.cc
@@ -17,7 +17,7 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-constexpr unsigned int VectorizedArray<double>::n_array_elements;
-constexpr unsigned int VectorizedArray<float>::n_array_elements;
+const unsigned int VectorizedArray<double>::n_array_elements;
+const unsigned int VectorizedArray<float>::n_array_elements;
 
 DEAL_II_NAMESPACE_CLOSE

